src/main/java/at/hannibal2/skyhanni/api/SkillAPI.kt

```diff
@@ -39,17 +40,31 @@ import kotlin.concurrent.fixedRateTimer
 import kotlin.time.Duration.Companion.seconds
 
 object SkillAPI {
-    private val patternGroup = RepoPattern.group("display.skilldisplay")
-    private val skillPercentPattern by patternGroup.pattern("skillpaternpercent", "\\+(?<gained>[\\d.,]+) (?<skillName>.+) \\((?<progress>[\\d.]+)%\\)")
-    private val skillPattern by patternGroup.pattern("skillpattern", "\\+(?<gained>[\\d.,]+) (?<skillName>\\w+) \\((?<current>[\\d.,]+)/(?<needed>[\\d.,]+)\\)")
-    private val skillMultiplierPattern by patternGroup.pattern("skillpatternmultiplier", "\\+(?<gained>[\\d.,]+) (?<skillName>.+) \\((?<current>[\\d.,]+)/(?<needed>[\\d,.]+[kmb])\\)")
-    private val skillTabPattern by patternGroup.pattern("skilltabpattern", "^§e§lSkills: §r§a(?<type>\\w+) (?<level>\\d+): §r§3(?<progress>.+)%\$")
-    private val maxSkillTabPattern by patternGroup.pattern("maxskilltabpattern", "^§e§lSkills: §r§a(?<type>\\w+) (?<level>\\d+): §r§c§lMAX\$")
-    val SPACE_SPLITTER = Splitter.on("  ").omitEmptyStrings().trimResults()
+    private val patternGroup = RepoPattern.group("api.skills.skilldisplay")
+    private val skillPercentPattern by patternGroup.pattern(
+        "skillpaternpercent",
```

Comment:
in line with the comment on the whole file id change this key to be more like
"skill.percent"